attestation-service/attestation-service/src/lib.rs

```diff
@@ -122,18 +159,30 @@ impl AttestationService {
             .await
             .map_err(|e| anyhow!("Generate reference data failed{:?}", e))?;
 
-        // Now only support using default policy to evaluate
         let evaluation_report = self
             .policy_engine
-            .evaluate(reference_data_map, tcb_json, None)
+            .evaluate(reference_data_map, tcb_json, policy_ids.clone())
             .await
             .map_err(|e| anyhow!("Policy Engine evaluation failed: {e}"))?;
 
+        let evaluation_reports: Vec<_> = evaluation_report
+            .into_iter()
+            .map(|(k, v)| {
+                json!({
+                    "policy-id": k,
```

Comment:
i see that policy-id is a `filename` i don't think that is future proof or secure.

security:
think of the relying party that wants to check that the token was checked against a certain policy. filenames can be changed.

future-proof:
filenames are tied to file abstractions, policies could be hosted elsewhere.

Can we store the hash of the policy in the attestation result instead? That is stable and unique.

Reply:
Nice idea. Let me add the digest of the policy too. BTW, the name of the policy might help the consumers to have some (though probably not trusted as you mentioned) contexts of the policy, so maybe we can keep it there?

Reply:
in that case i'd prefer uuid + hash than name and hash